test/cpp/client/credentials_test.cc

```diff
@@ -670,6 +679,22 @@ TEST_F(CredentialsTest, LoadTlsChannelCredentials) {
   GPR_ASSERT(channel_credentials != nullptr);
 }
 
+// This test demonstrates how the TLS credentials will be used to create
```

Comment:
Can we have a channel-counterpart for this test as well?

Reply:
You probably need to re-sync to upstream in order to do that due to recent changes by Microsoft.